tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "sourceMap": true,
+    "module": "commonjs",
```

Comment:
This should probably be `es6`. @hswolff that sound right?

Reply:
This is how TypeScript transpiles your code. Since we're not transpiling we can probably omit, or what we did in Cloud is have this, and target be `esnext` to effectively turn off transpiling.